app/validators/special_character_validator.rb

```diff
@@ -1,17 +1,13 @@
 class SpecialCharacterValidator < ActiveModel::EachValidator
 
   def validate_each(record, attribute, value)
-    return if value.nil?
+    return if value.nil? || value.blank?
```

Comment:
Isn't the responsibility of checking blank down to the 'allow_blank: true' option in the validates call ?  It doesnt seem right writing a validator that allows blank without you telling it to ?